aea/protocols/dialogue/base.py

```diff
@@ -1855,9 +1865,7 @@ def _complete_dialogue_reference(self, message: Message) -> None:
             incomplete_dialogue_reference, message.sender, self.self_address,
         )
 
-        if self._dialogues_storage.is_dialogue_present(
```

Comment:
why the change?

Reply:
Now we have this _{incomplete_label: None}_ entries, _incomplete_dialogue_label_ will always be in the incomplete mapping, so if we keep this conditional we would never enter it.